modules/helfi_calculator/src/Form/CalculatorSettings.php

```diff
@@ -83,27 +116,26 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $form['#prefix'] = '<div class="layer-wrapper"><h2>' . t('Available calculators') . '</h2>';
     $form['#suffix'] = '</div>';
 
-    $calculators = $settings->get('calculator_settings');
+    $calculators = $settings->get('calculators');
 
     foreach ($calculators as $key => $value) {
-      $title = ucfirst(str_replace("_", " ", $key));
-
-      $form['calculator_settings'][$key] = [
+      $a = $this->getCalculatorLabel($key);
```

Comment:
Is this variable used somewhere? If it is, it should be renamed, otherwise it should be removed.

Reply:
Nice catch! Removing - was just used for debugging :)